addons/mail/static/src/models/partner/partner.js

```diff
@@ -514,6 +511,9 @@ function factory(dependencies) {
                 'name',
             ],
         }),
+        threadMembers: one2many('mail.thread_member', {
```

Comment:
This name is a bit confusing on partner, especially if there is the same field name on thread.

Do we actually need it here?